src/components/assets/icons/SSIEntityIcon/index.tsx

```diff
@@ -1,29 +1,40 @@
+import { Avatar } from '@rneui/themed'
```

Comment:
i rather not use third party packages for this. making a view with a style that makes it a circle does not require anything

Reply:
fixed